upgrade/helpers/tools.py

```diff
@@ -95,6 +96,45 @@ def host_pings(host, timeout=15):
             time.sleep(5)
 
 
+def host_ssh_check(host, timeout=15):
```

Comment:
Rename to something informative
e.g host_ssh_availability_check or something better